site/content/docs/5.3/migration.md

```diff
@@ -138,6 +138,9 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
 
 - **Typography**
   - We no longer set a color for `$headings-color-dark` or `--bs-heading-color` for dark mode. To avoid several problems of headings within components appearing the wrong color, we've set the Sass variable to `null` and added a `null` check like we use on the default light mode.
+  - <span class="badge bg-warning">Breaking</span> Some headings font size values have been changed to be compliant with the design guidelines. Please check it has no impact on your websites.
+    - `<h3>` values are now `24px` for desktop and tablet breakpoints and `18px` for mobile breakpoint.
```

Comment:
For these 2 sentences, could you double-check, we'd like to mention only what has changed; in terms of values and not in terms of documentation.

If I refer to the documentation only of the current main branch (which is probably wrong), `<h3>` being `24px` for desktop is not new, and `<h5>` being `18px` for desktop neither, for example.

Reply:
After spec review, we'll need to fix font size.